data/heliarch intro.txt

```diff
@@ -0,0 +1,1568 @@
+			`	"Unearthed some useful information, we have," Iplora tells her. "Essential to the mission's success, Captain <last>'s cooperativeness was."`
+			`	"No doubt of that, I have. To the report, looking forward I am."`
+			`	They thank you once again for helping, and then walk to the Heliarch-only parts of the ringworld.`
+				accept
+			label evidence1
+			`The Heliarch elder, Iekie, greets you and the agents when you return, and gives you your payment of 900000 credits.`
+			`	"Unearthed some useful information, we have," Iplora tells her. "Essential to the mission's success, Captain <last>'s cooperativeness was."`
+			`	"No doubt of that, I have. To the report, looking forward I am."`
+			`	They then get closer to her, and Iplora whispers something, handing her a separate report card, written differently than the Coalition languages that you've seen. It must be some type of code.`
+			`	Iekie turns to you once more, saying, "Quite a find, this is. Done well, you have, Captain."`
+				goto finalthanks
+			label evidence2
+			`The Heliarch elder, Iekie, greets you and the agents when you return, and gives you your payment of 900000 credits.`
+			`	"Unearthed some useful information, we have," Iplora tells her. "Essential to the mission's success, Captain <last>'s cooperativeness was."`
+			`	"No doubt of that, I have. To the report, looking forward I am."`
+			`	They then get closer to her, and Iplora whispers something, handing her two separate report cards, written differently than the Coalition languages that you've seen. It must be some type of code.`
+			`	Iekie turns to you once more, saying, "Most useful to our investigation, these findings will prove. Excellent work, Captain."`
+				goto finalthanks
+			label evidence3
+			`The Heliarch elder, Iekie, greets you and the agents when you return, and gives you your payment of 900000 credits.`
+			`	"Unearthed some useful information, we have," Iplora tells her. "Essential to the mission's success, Captain <last>'s cooperativeness was."`
+			`	"No doubt of that, I have. To the report, looking forward I am."`
+			`	They then get closer to her, and Iplora whispers something, handing her three separate report cards, written differently than the Coalition languages that you've seen. It must be some type of code.`
+			`	Iekie turns to you once more, saying, "Enviable, your prowess at investigation is, <last>. Outstanding work, Captain."`
+				goto finalthanks
+			label finalthanks
+			`	They thank you once again for helping, and then walk to the Heliarch-only parts of the ringworld.`
+
+
+
+mission "HI2-Bonus 1"
+	invisible
+	landing
+	to offer
+		has "Heliarch Investigation 2: done"
+		"resistance evidence" == 1
+	source
+		government "Coalition" "Heliarch"
+	on offer
+		payment 300000
+		conversation
+			`Coming in for a landing, you receive a message from the the Heliarch elder Iekie.`
```

Comment:
`You receive a message from the Heliarch elder Iekie as you go in for a landing.`